openai/openai.go

```diff
@@ -86,6 +92,38 @@ type ChatCompletionChunk struct {
 	Choices           []ChunkChoice `json:"choices"`
 }
 
+type CompletionRequest struct {
+	Model            string   `json:"model"`
+	Prompt           string   `json:"prompt"` // openai also accepts a list of strings if we want to do that
```

Comment:
Great catch. We can leave this as a TODO for now – possible to create an open issue for it and link it here:

```
TODO (https://github.com/ollama/ollama/issues<id>): support []string, []int and [][]int
```

or similar